packages/Avatar/README.md

```diff
@@ -22,12 +22,12 @@ npm install @paprika/avatar
 
 ### Avatar
 
-| Prop            | Type               | required | default  | Description                                           |
-| --------------- | ------------------ | -------- | -------- | ----------------------------------------------------- |
-| children        | node               | false    | null     | Avatar content. It can be initial as a string or icon |
-| backgroundColor | string             | false    | null     | Background color of the Avatar                        |
-| color           | string             | false    | null     | Color for the initial or icon                         |
-| size            | ShirtSizes.LIMITED | false    | "medium" | Size of Avatar                                        |
+| Prop            | Type                       | required | default      | Description                                           |
+| --------------- | -------------------------- | -------- | ------------ | ----------------------------------------------------- |
+| children        | node                       | false    | null         | Avatar content. It can be initial as a string or icon |
+| backgroundColor | string                     | false    | null         | Background color of the Avatar                        |
+| color           | string                     | false    | null         | Color for the initial or icon                         |
+| size            | [types.SMALL,types.MEDIUM] | false    | types.MEDIUM | Size of Avatar                                        |
```

Comment:
If we want to render `[Avatar.types.size.SMALL,Avatar.types.size.MEDIUM]` in the README.md, we'll need to use `size: PropTypes.oneOf([Avatar.types.size.SMALL, Avatar.types.size.MEDIUM]),` in propTypes definition. e.g. here: https://github.com/acl-services/paprika/pull/664/files#diff-1295811f8a1709758fc99b9871b2aba9R15
However since we usually put the propTypes definition at the beginning of files, if we make this change, there'll be some lint errors. Maybe need yours input @nahumzs 🙇‍♀️

Reply:
yeah, I thought about that and then forgot about it, well I think with this PR and the one of Oscar and Karen we will have a way for avoiding to come to the source file to read the props. So theoretically we could move the props definition from the top to the bottom, or just relay on javascript and knowing that the compiler will move all those declaration to the top automatically and ignore the linter. (which I think is the easier one and really not harmful) 

What do you think?

Reply:
I'd prefer the second way too, combined with `/* eslint-disable no-use-before-define */` and `/* eslint-enable no-use-before-define */`